### PR TITLE
'Add test for re-registration of host with expired SCA cert

### PR DESCRIPTION
**Pull request by synkd**
Closed-Loop BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1949353

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

**Review by JacobCallahan — APPROVED** [review]
Clean!

File: tests/foreman/api/test_subscription.py [hunk 1]
```diff
@@ -355,3 +355,59 @@ def test_positive_candlepin_events_processed_by_stomp(rhel7_contenthost, functio
...
+        3. Unregister the content host
+        4. Enable and synchronize a repository
+        5. Re-register the host using the same activation key as in step 3 above
+
+    :expectedresults: the host is re-registered successfully and its SCA entitlement
+                      certificate is refreshed
+
+    :Assignee: dsynk
+
+    :BZ: 1949353
```
Comment (mshriver) [inline tests/foreman/api/test_subscription.py:379]: This BZ has customer cases attached, please set `CustomerScenario: true` token too

Comment (synkd) [reply tests/foreman/api/test_subscription.py:381]: Done.

**Review by mshriver — APPROVED** [review]

**Comment by synkd** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_expired_SCA_cert_handling' tests/foreman/

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_expired_SCA_cert_handling' tests/foreman/

**Comment by mshriver** [comment]
Approved as the content looks strong, but @synkd is going to investigate test failure flakyness. Test has passed/failed in local env.

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_expired_SCA_cert_handling' tests/foreman/

File: tests/foreman/api/test_subscription.py [hunk 2]
```diff
@@ -355,3 +355,61 @@ def test_positive_candlepin_events_processed_by_stomp(rhel7_contenthost, functio
...
+        1. Import an SCA-enabled manifest
+        2. Register a content host to the Default Organization View using an activation key
+        3. Unregister the content host
+        4. Enable and synchronize a repository
+        5. Re-register the host using the same activation key as in step 3 above
+
+    :expectedresults: the host is re-registered successfully and its SCA entitlement
+                      certificate is refreshed
+
+    :CustomerScenario: true
```
Comment (tstrych) [inline tests/foreman/api/test_subscription.py:377]: Everywhere in the code lowercase `CustomerScenario` is used, not sure if it matter.

Comment (mshriver) [reply tests/foreman/api/test_subscription.py:377]: Either works, testimony only enforces case sensitivity of the values, not the tokens themselves.

**Comment by synkd** [comment]
The test failure that I encountered appears to be a known issue that should be fixed in an upcoming snap: https://bugzilla.redhat.com/show_bug.cgi?id=1962947. I will run the check again after we receive a snap that includes the fix.

**Comment by synkd** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_expired_SCA_cert_handling' tests/foreman/

**Pull request merged**